cf/data/dask_utils.py

```diff
@@ -594,3 +593,40 @@ def cf_dt2rt(a, units):
 
     """
     return dt2rt(a, units_out=units, units_in=None)
+
+
+def cf_units(a, from_units, to_units):
+    """Convert array values to have different equivalent units.
+
+    .. versionadded:: TODODASK
+
+    .. seealso:: `cf.Data.Units`
+
+    :Parameters:
+
+        a: `numpy.ndarray`
+            The array.
+
+        from_units: `Units`
+            The existing units of the array.
+
+        to_units: `Units`
+            The units that the array should be converted to. Must be
+            equivalent to *from_units*.
+
+    :Returns:
+
+        `numpy.ndarray`
+            An array containing values in the new units.
```

Comment:
It seems like the `dtype` can change (e.g. `int` to `float`) in this process, it might be worth noting that here (just to continue with the thoroughness of the documentation you're adding overall!)?

Reply:
Good idea: https://github.com/NCAS-CMS/cf-python/pull/405/commits/cb617cc771f89cbd9f69107c583de3b357fb2f35